.gitignore

```diff
@@ -35,3 +35,15 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+# IntelliJ
+.idea
+*.iml
+
+# Eclipse
+.project
+.classpath
+.settings
```

Comment:
This is a minor nit, but I think best practice is to keep editor and other developer-specific settings in your global `.gitignore` file, and only project-specific ignores in the project file. Here is a decent article covering this topic https://zellwk.com/blog/gitignore/. You might also want to check out github's comprehensive gitignore repository with complete editor/application specific patterns here https://github.com/github/gitignore.